docs/datasync/delta-table.md

```diff
@@ -0,0 +1,208 @@
+---
+id: delta-table
+title: Delta Tables with server-side Conflict resolution
+sidebar_label: Setting up server-side Conflict Resolution
+---
+
+This is a more involved strategy for data synchronization that includes server-side conflict resolution
+
+## Setup
+
+Feel free to grab the official Graphback template for this strategy [here](https://GitHub.com/aerogear/graphback/tree/master/templates/ts-apollo-mongodb-datasync-backend). Otherwise, It is fairly easy to set this up with the official Graphback template for MongoDB (found [here](https://GitHub.com/aerogear/graphback/tree/master/templates/ts-apollo-mongodb-backend)).
+
+### Annotate the required models
+
+Add the `@datasync` annotation to your model(s) in your GraphQL SDL found in the `model` folder:
+
+```graphql {3}
+""" 
+@model
+@datasync
+"""
+type Comment {
+  _id: GraphbackObjectID!
+  text: String
+  description: String
+}
+```
+
+This part retains the features from [Soft-Deletes](soft-delete.md) with the addition of some extras:
+
+1. A `_version` field in the model type used for fetching base document in conflict resolution.
+2. A `_version` field in the corresponding Mutation Input Type.
+
+The model type then, becomes:
+```graphql {9,10}
+""" 
+@model
+@datasync
+"""
+type Comment {
+  _id: GraphbackObjectID!
+  text: String
+  description: String
+  _lastUpdatedAt: GraphbackTimestamp
+  _version: Int
+}
+```
+
+And the Mutation Input type for it would be:
+```graphql {5}
+input MutateCommentInput {
+  _id: GraphbackObjectID!
+  text: String
+  description: String
+  _version: Int!
+}
+```
+
+An example mutation can be found at the end of this page.
+
+### Modify the template to support Data Synchronization
+
+In the [`src/index.ts`](https://github.com/aerogear/graphback/blob/master/templates/ts-apollo-mongodb-backend/src/index.ts) file of the template, use  `createDataSyncAPI` instead of `buildGraphbackAPI`:
```

Comment:
```suggestion
In the [`src/index.ts`](https://github.com/aerogear/graphback/blob/templates-release/templates/ts-apollo-mongodb-backend/src/index.ts) file of the template, use  `createDataSyncAPI` instead of `buildGraphbackAPI`:
```